platforms/yttrex/backend/bin/parser.ts

```diff
@@ -96,8 +96,19 @@ const run = async (): Promise<void> => {
         ...parserConfig,
         errorReporter: (e: HTMLSource) => {
           const entryNature = e.html.nature.type ?? 'failed';
+
+          const fixtureFolderPath = path.resolve(
+            process.cwd(),
+            '__tests__/fixtures/htmls'
+          );
+
+          // ensure fixtures folder path exists
+          if (!fs.existsSync(fixtureFolderPath)) {
+            fs.mkdirSync(fixtureFolderPath);
+          }
+
```

Comment:
After the folder creation, I'd love to have also a loop to create the other needed folders.
As far as I know, they are home,search,video,channel and unknown

So I'd like something like

```js
const subfolders = ["home","search","video","channel","unknown"]
for (const sf of subfolders){
    const fullPath = path.resolve(fixtureFolderPath, sf));
    if (!fs.existsSync(fullPath))
                 fs.mkdirSync(fullPath);
}
```

Reply:
I added the `entryNature` directly to `fixtureFolderPath`